src/Database/Query/CacheableQueryBuilder.php

```diff
@@ -81,8 +86,16 @@ public function __construct(
         $this->modelIdentifier = $cacheableProperties['identifier'] ?? null;
         $this->ttl = $cacheableProperties['ttl'] ?? null;
         $this->prefix = $cacheableProperties['prefix'] ?? null;
+        $this->logChannel = Arr::get($cacheableProperties, 'logging.channel', null);
         $this->logEnabled = Arr::get($cacheableProperties, 'logging.enabled', false);
-        $this->logLevel = Arr::get($cacheableProperties, 'logging.level', null);
+        $this->logLevel = Arr::get($cacheableProperties, 'logging.level', 'debug');
+
+        if ($this->logChannel == null || $this->logChannel == 'default') {
```

Comment:
Hey @Sleepy4k , sorry had a lot to do.

Please only use `null` as default and no special string case. Maybe someone will call a channel later default, which is something completely different.
`null` should be defaultDriver, the rest the according channel.

Thanks

Reply:
i am sorry, i forgot to remove it